modules/qtwidgets/src/processors/processordockwidgetqt.cpp

```diff
@@ -52,7 +52,7 @@ ProcessorDockWidgetQt::ProcessorDockWidgetQt(Processor* p, const QString& title,
     ivec2 dim = ProcessorWidget::getDimensions();
     ivec2 pos = ProcessorWidget::getPosition();
 
-    setWindowTitle(QString::fromStdString(processor_->getIdentifier()));
+    setWindowTitle(QString::fromStdString(processor_->getDisplayName()));
```

Comment:
use `utilqt::fromQString`
Note that the display name is not necessarily unique!